modules/installation-configuration-parameters.adoc

```diff
@@ -639,11 +639,11 @@ ifndef::aws,bare,ibm-z,ibm-power,azure,ibm-power-vs[]
 |String
 endif::aws,bare,ibm-z,ibm-power,azure,ibm-power-vs[]
 
-ifdef::aws,bare,azure[]
+ifdef::aws,bare,azure,gcp[]
 |`controlPlane.architecture`
```

Comment:
it seems double `controlPlane.architecture` field in the doc [installation-config-parameters-gcp](https://62253--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installation-config-parameters-gcp#installation-configuration-parameters-optional_installation-config-parameters-gcp)